CMakeLists.txt

```diff
@@ -190,6 +200,7 @@ ADD_COMPILE_DEFINITIONS("XNN_ENABLE_AVXVNNI=$<BOOL:${XNNPACK_ENABLE_AVXVNNI}>")
 ADD_COMPILE_DEFINITIONS("XNN_ENABLE_AVX256SKX=$<BOOL:${XNNPACK_ENABLE_AVX256SKX}>")
 ADD_COMPILE_DEFINITIONS("XNN_ENABLE_AVX256VNNI=$<BOOL:${XNNPACK_ENABLE_AVX256VNNI}>")
 ADD_COMPILE_DEFINITIONS("XNN_ENABLE_AVX256VNNIGFNI=$<BOOL:${XNNPACK_ENABLE_AVX256VNNIGFNI}>")
+ADD_COMPILE_DEFINITIONS("XNN_ENABLE_AVX512VNNIGFNI=$<BOOL:${XNNPACK_ENABLE_AVX512VNNIGFNI}>")
```

Comment:
This is good to have but the macros apply in a few more places.  They are mainly for the benefit of bazel that doesnt have a compiler version but can disable instruction sets, so there are command line flags in the BUILD file that get passed as macros for the gemm-config.
Its more important for newer instruction sets like AVX10, that are not supported by older compilers.
But its also handy as a way to compiler benchmarks with an instruction set disabled to measure impact.